src/Sarif.UnitTests/Sarif.UnitTests.csproj

```diff
@@ -13,6 +13,11 @@
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
+  <ItemGroup>
+    <None Remove="Transformers\TestLogs\v1\CodeFlows.sarif" />
+    <None Remove="Transformers\TestLogs\v2\CodeFlows.sarif" />
+  </ItemGroup>
```

Comment:
Just:
```
<ItemGroup>
  <None Remove="Transformers\TestsLogs\**\*.sarif" />
</ItemGroup>
```

Reply:
This was added by VS, so hopefully this wildcard version will remain when we add more files.

---
In reply to: [205148961](https://github.com/Microsoft/sarif-sdk/pull/966#discussion_r205148961) [](ancestors = 205148961)